Bindings/Java/Matlab/Hopper_Device/README.txt

```diff
@@ -0,0 +1,40 @@
+MATLAB Example: *Hopper Device*
+===============================
+
+This is the MATLAB version of the Hopper Device example (there is also a C++
+version). The example demonstrates some of the new features of the OpenSim 4.0
+API. The Component architecture allows us to join sub-assemblies to form larger
+Models, with information flowing between Components via Inputs, Outputs, and
+Sockets. For more information, please refer to the Doxygen documentation.
```

Comment:
I think it is confusing to associate Sockets with Inputs and Outputs in the context of information flow. A Socket is dependency on another Component whereas an Input is a a dependency on an Output (which is component generated data/information).